lib/chef/mixin/common_helpers.rb

```diff
@@ -0,0 +1,37 @@
+#
+# Author:: Kapil Chouhan (<kapil.chouhan@msystechnologies.com>)
+# Copyright:: Copyright 2008-2018, Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+class Chef
+  module Mixin
+    module CommonHelpers
```

Comment:
This should go into one of the Chef::DSL mixins that are already there for this purpose, rather than inventing a new one:

https://github.com/chef/chef/blob/master/lib/chef/dsl/universal.rb#L27-L44

Reply:
(probably universal since that is the one included in Chef::Resource)